.github/workflows/build-images.yml

```diff
@@ -285,7 +285,7 @@ jobs:
       uses: spilchen/switch-case-action@v2
       id: v2_vertica_image
       with:
-        default: ghcr.io/${{ env.OWNER_LC }}/vertica-k8s:${{ github.sha }}-v2
+        default: verticadocker/vertica-k8s-private:latest-master
         conditionals-with-values: |
           ${{ inputs.v2_vertica_image != '' }} => ${{ inputs.v2_vertica_image }}
           ${{ github.event_name == 'pull_request' }} => vertica-k8s:kind-v2
```

Comment:
I don't think you applied everything from my suggestion. This line for instance should be removed. For PR's we want to use the default name set at 288. There was also a change needed in another file. Can you review the change I suggested again?

Reply:
@spilchen I already tried your suggestion, but it looks like that we can't docker save the latest-master image 
```
Run docker save verticadocker/vertica-k8s-private:latest-master > v2-vertica-image.tar
Error response from daemon: reference does not exist
Error: Process completed with exit code 1.
```